code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -830,3 +830,10 @@
 
 /datum/species/proc/can_double_fireman_carry()
 	return FALSE
+
+/datum/species/proc/has_stamina_for_pushup(var/mob/living/carbon/human/human)
```

Comment:
I think i'd rather this be a generic has_stamina proc for X amount of stamina drain, but w.e